README.md

```diff
@@ -29,50 +31,50 @@ As of Dec, 22, 2021, this starter is still far from covering most basic cases fo
 
 0. **Clone this repo**
 
-    Get up and running by cloning this repo.
+   Get up and running by cloning this repo.
 
-    ```shell
-    # Clone this repo into your machine
-    npx degit vtex-sites/base.store awesome.store
-    ```
+   ```shell
+   # Clone this repo into your machine
+   npx degit vtex-sites/base.store awesome.store
+   ```
 
-1.  **Install dependencies**
+1. **Install dependencies**
 
-    Install dependencies with yarn
+   Install dependencies with yarn
 
-    ```shell
-    cd awesome.store/
-    yarn
-    ```
+   ```shell
+   cd awesome.store/
+   yarn
+   ```
 
-2.  **Setup store.config.js**
+2. **Setup store.config.js**
 
-    Choose the ecommerce platform provider of your choice in the `store.config` file and set the corresponding options. For instance, to connect to the VTEX platform on the store `fashioneurope`:
+   Choose the ecommerce platform provider of your choice in the `store.config` file and set the corresponding options. For instance, to connect to the VTEX platform on the store `fashioneurope`:
 
-    ```js
-    module.exports = {
-      platform: 'vtex',
+   ```js
+   module.exports = {
+     platform: 'vtex',
 
-      api: {
-        storeId: 'fashioneurope'
-        environment: 'vtexcommercestable'
-      }
-    }
-    ```
+     api: {
+       storeId: 'fashioneurope'
+       environment: 'vtexcommercestable'
+     }
+   }
+   ```
 
-3.  **Start developing**
+3. **Start developing**
 
-    Navigate into your new site’s directory and start it up.
+   Navigate into your new site’s directory and start it up.
 
-    ```shell
-    yarn develop
-    ```
+   ```shell
+   yarn develop
+   ```
 
-1.  **Open the source code and start editing!**
+4. **Open the source code and start editing!**
 
-    Your site is now running at `http://localhost:8000`!
+   Your site is now running at `http://localhost:8000`!
 
-> Note: You'll also see a second link: _`http://localhost:8000/___graphql`_. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.com/docs/tutorial/part-4/#use-graphiql-to-explore-the-data-layer-and-write-graphql-queries). Also, if you want to query for dynamic data (allProducts, allCollections, search etc), you can use tools like [GraphQL Playground](https://github.com/graphql/graphql-playground) and configure it to query _`http://localhost:8000/api/graphql`_
+> Note: You'll also see a second link: \_`http://localhost:8000/___graphql`_. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.com/docs/tutorial/part-4/#use-graphiql-to-explore-the-data-layer-and-write-graphql-queries). Also, if you want to query for dynamic data (allProducts, allCollections, search etc), you can use tools like [GraphQL Playground](https://github.com/graphql/graphql-playground) and configure it to query _`http://localhost:8000/api/graphql`\_
```

Comment:
Why did you add the `\`? This bit looks weirder now.

Before|After
-|-
![CleanShot 2022-01-17 at 12 34 37](https://user-images.githubusercontent.com/381395/149798631-bb34e38e-a7ed-4d9b-bb25-177468e70079.png)|![CleanShot 2022-01-17 at 12 33 32](https://user-images.githubusercontent.com/381395/149798664-d6f66a25-9836-4829-a228-6da72631feef.png)

Reply:
This was an automatic fix by our Lint. Should I move it back?

Reply:
That's weird! I tried changing this file but I didn't have the same result, the linter didn't touch this line. There must be some misconfiguration. 
😕 

I suggest reverting this change specifically.

Reply:
@filipewl could you try to do it? When I save the file que bar is added automatically :(

Reply:
the same thing is happening here :(

Reply:
There's some tooling conflict that is adding these `\` back... To simplify things I [removed](https://github.com/vtex-sites/base.store/pull/224/commits/214cf00bdba86f3a07159a059cceaf9746a37b9b) both `\` and the _italiczation_ via `_`.